shared/constants/keyring.ts

```diff
@@ -0,0 +1,17 @@
+import { HardwareKeyringTypes } from './hardware-wallets';
+
+/**
+ * These are the keyrings that are managed entirely by MetaMask.
+ */
+export enum InternalKeyringTypes {
```

Comment:
While we are fixing this one, should we also depluralize these? We have both plural enums and non plural enums at the moment and we should probably choose

Reply:
Good point

Reply:
Done here: ac73f8236665fd2f7e101fe0891e1b2ad46741b7